src/ptychography/reconstruction/ssb.py

```diff
@@ -73,7 +74,7 @@ def rotate_sysx(real_sy, real_sx, cos_angle, sin_angle):
 
 
 def generate_masks(reconstruct_shape, mask_shape, dtype, wavelength, dpix, semiconv,

```

Comment:
What is the unit of `cutoff`? I think `generate_masks` could use a docstring, also update of the UDF docstring for `cutoff` parameter.

Reply:
Something else: the `wavelength` parameter shadows the imported `wavelength` function; one of these should be renamed